openedx_external_enrollments/api/v0/views.py

```diff
@@ -383,6 +383,18 @@ def _get_program_of_interest_data(self, data, order_lines):
                     openedx_user.username,
                     request_time.strftime("%Y-%m-%d-%H:%M:%S"),
                 )
+                program_of_interest["Lead_Source"] = data.get(
```

Comment:
This code is required either for programs or "single_course" salesforce enrollments, therefore it's scope should be wider. Let's unindent those lines to apply the code regardless of `if-else` condition. @andrey-canon

Reply:
Since there is no a specific requirement to override programs values and I have no idea how programs works or why that was created, these changes shouldn't override  a database value that looks more important, however I'm going to ask

Reply:
The Scott answer was 'check in with Diego', I don't know the business logic behind this, so addressed

Reply:
The business logic was: Get all UTM parameters either from other course settings (in case of a single enrollment) or from a program table (defined within this plugin), before calling salesforce API. Those two cases are represented in this piece of code as "if-else" clause. Since we need to override the behavior all the times the code in charge of the update related to any "Source" param must be in the outer scope, as you already modified it. I'm approving this PR @andrey-canon

Reply:
We will get rid of redundant DB configurations for both cases. Just for sanity @andrey-canon